packages/http/src/mocker/negotiator/__tests__/NegotiatorHelpers.spec.ts

```diff
@@ -527,7 +527,7 @@ describe('NegotiatorHelpers', () => {
       expect(actualOperationConfig).toBe(fakeOperationConfig);
     });
 
-    it('given no default contents should throw', () => {
+    it('given no default contents should return default response', () => {
```

Comment:
Sorry to quibble on the English here, but we are using the word "default" twice to mean two different things. 

> when no default response return text/plan with empty body

Reply:
Done